src/java/picard/fastq/BamToBfq.java

```diff
@@ -38,7 +38,18 @@
  * @author ktibbett@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = "Create BFQ files from a BAM file for use by the Maq aligner.",
+        usage = "Bfq is a binary version of the fastq file.  This tool creates BFQ files from a BAM file for use by" +

```

Comment:
Man, this is an antique. I wonder if it even makes any sense to maintain this tool. Nobody should use this anymore.